index.d.ts

```diff
@@ -47,6 +47,7 @@ export declare type RecaptchaProps = {
     recaptchaDomain?: string;
     gstaticDomain?: string;
     hideBadge?: boolean;
+    action?: string;
```

Comment:
Doubt: Does this field accept any string?

Reply:
@douglasjunior According to [this](https://cloud.google.com/recaptcha-enterprise/docs/actions), more expressive documentation action can be any string